### PR TITLE
Add Agile Timeboxing Question

### PR DESCRIPTION
**Pull request by mukul-kr**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

#103

### CODE REVIEW

File: src/data/agile-quiz.ts [hunk 1]
```diff
@@ -175,6 +175,17 @@ const agileQuiz = [
       "Planning poker is an estimation and planning technique in the Agile development model. It is used to estimate the development effort required for a user story or a feature.",
     Link: "https://www.freecodecamp.org/news/complete-guide-to-agile-methodology/",
   },
+  {
+    Question:
+      "What is Time-Boxing?",
```
Comment (jdwilkin4) [inline src/data/agile-quiz.ts:180]:
```suggestion
      "In Agile, what is Timeboxing?",
```

File: src/data/agile-quiz.ts [hunk 2]
```diff
@@ -175,6 +175,17 @@ const agileQuiz = [
...
+    Answer: "Keeping to the Schedule",
```
Comment (jdwilkin4) [inline src/data/agile-quiz.ts:181]:
```suggestion
    Answer: "set amount of time for an activity to occur",
```

File: src/data/agile-quiz.ts [hunk 3]
```diff
@@ -175,6 +175,17 @@ const agileQuiz = [
...
+    Distractor1: "It's a new Agile framework",
+    Distractor2: "Not related to Agile",
+    Distractor3: "It's a way to write code faster",
+    Explanation:
+      "In Agile principles, timeboxing allocates a fixed and maximum unit of time to an activity, called a timebox, within which planned activity takes place. It is used by Agile principles-based project management approaches and for personal time management.",
```
Comment (jdwilkin4) [inline src/data/agile-quiz.ts:186]: Please avoid using direct quotes from external sources like Wikipedia.
It is always to write it in your own words 👍 

```suggestion
      "In Agile, Timeboxing is a set amount of time for a planned activity to take place. This technique is often used in project management to help with productivity for projects. ",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by mukul-kr** [comment]
I have resolved the requested changes. Can you please review it?

**Review by jdwilkin4 — APPROVED** [review]
Congrats on another contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**